.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/bin/python3"

```

Comment:
Should we have this file? It looks like it's a hidden one so not sure we should be committing it

Reply:
my bad, it shouldn't be here